.circleci/config.yml

```
@@ -0,0 +1,38 @@
+version: 2.1
+
+executors:
+  go:
+    docker:
+      - image: circleci/golang:1.12.1
+
+commands:
+  report_coverage:
```

I think `upload` makes the step more explicit
```suggestion
  upload_coverage:
```